---
layout: page
title: Dr. Xiaofei Yang
---


<h1 class="header center blue-text">{{ page.title | escape }}</h1>

<div class = "row">
    <div class = "row">
            <div class = "col s12 m12">
                <div id = "bio">
                    <h5>Biograph</h5>
                    <p>杨晓飞，西安交通大学讲师，电信学院，硕士生导师。2011年本科毕业于西安电子科技大学软件学院，2016年毕业于西安电子科技大学计算机学院，获得计算机应用技术博士学位。2017年加入西安交通大学“叶凯青年科学家工作室”，2018年10月赴美国杰克逊实验室基因组医学研究中心进行访问学习，为期一年。目前研究方向有生物信息学，植物基因组学，癌症基因组学，表观遗传学，基因组结构变异检测，非编码RNA功能研究等。研究成果发表在Science, Genome Biology, Briefings in bioinformatics, Human molecular genetics, Briefings in functional genomics, Nucleic acids research, 科学通报等多个国际主流期刊发表学术论文14篇，累计引用次数368次，h指数为9。目前主持国家自然科学基金青年项目1项，中国博士后面上资助项目1项，陕西省博士后资助项目一项，以子课题负责人身份参与科技部重大专项1项。杨晓飞，西安交通大学讲师，电信学院，硕士生导师。2011年本科毕业于西安电子科技大学软件学院，2016年毕业于西安电子科技大学计算机学院，获得计算机应用技术博士学位。2017年加入西安交通大学“叶凯青年科学家工作室”，2018年10月赴美国杰克逊实验室基因组医学研究中心进行访问学习，为期一年。目前研究方向有生物信息学，植物基因组学，癌症基因组学，表观遗传学，基因组结构变异检测，非编码RNA功能研究等。研究成果发表在Science, Genome Biology, Briefings in bioinformatics, Human molecular genetics, Briefings in functional genomics, Nucleic acids research, 科学通报等多个国际主流期刊发表学术论文14篇，累计引用次数368次，h指数为9。目前主持国家自然科学基金青年项目1项，中国博士后面上资助项目1项，陕西省博士后资助项目一项，以子课题负责人身份参与科技部重大专项1项。</p>
                    </div>
                <div id = "edu">
                    <h5>Education</h5>
                    <table style="border:none;background:none;">
                    <tr> 
                        <th>Ph.D.</th>
                        <th>Computer Science and Technology</th>
                        <th>Xidian University</th>
                        <th>2016.12</th>
                    </tr>
                    <tr style="border:none;background:none;">
                        <th>B.A.</th>
                        <th>Software Engineering</th>
                        <th>Xidian University</th>
                        <th>2011.07</th>
                    </tr>
                    </table>
                </div>
                <div id = "work">
                    <h5>Working Experience</h5>
                    <table style="border:none;background:none;">
                    <tr>
                        <th>Assistant Professor</th>
                        <th>Xi’an Jiaotong University</th>
                        <th>2017.06-Present</th>
                    </tr> 
                    <tr>
                         <th>Visiting Scholarr</th>
                         <th>The Jackson Laboratory for Genomic Medicine</th>
                         <th>2018.10 – Present</th>
                    </tr>  
                    </table>
                </div>
                <div id = "researches">
                     <h5>Research Focus</h5>
                     <img src ="images/Yang Xiaofei_research_1.png" alt = " " height="200" align="left"/>
                     <b>The opium poppy genome and morphinan production. </b>The opium poppy genome reveals gene duplication, rearrangement and fusion events that led to a BIA-gene cluster for noscapine and morphine production. We report a draft of the opium poppy genome, with 2.72Gb assembled into 11 chromosomes with contig N50 and scaffold N50 of 1.77Mb and 204Mb, respectively. Synteny analysis suggests a whole genome duplication at approximately 7.8 million years ago (MYA) and ancient segmental or whole genome duplication(s) that occurred before the Papaveraceae-Ranunculaceae divergence 110 MYA. Syntenic blocks representative of phthalideisoquinoline and morphinan components of a benzylisoquinoline alkaloid cluster of 15 genes provides insight into how it evolved. Paralog analysis identified P450 and oxidoreductase genes that combined to form the STORR gene fusion essential for morphinan biosynthesis in opium poppy. Thus gene duplication, rearrangement and fusion events have led to evolution of specialized metabolic products in opium poppy.<br><br>                  
             <img src ="images/Yang Xiaofei_research_2.png" alt = " " height="400" align="right"/> <b>Pan-cancer analysis reveals cancer common and specific DNA methylation patterns.</b> We investigate cancer common and specific DNA methylation patterns among 5480 DNA methylation profiles of 15 cancer types from TCGA. We first define differentially methylated CpG sites (DMCs) in each cancer and then identify 5450 hyper- and 4433 hypomethylated pan-cancer DMCs (PDMCs). Intriguingly, three adjacent hypermethylated PDMC constitute an enhancer region, which potentially regulates two tumor suppressor genes BVES and PRDM1 negatively. Moreover, we identify six distinct motif clusters, which are enriched in hyper- or hypomethylated PDMCs and are associated with several well-known cancer hallmarks. We also observe that PDMCs relate to distinct transcriptional groups. Additionally, 55 hypermethylated and 7 hypomethylated PDMCs are significantly associated with patient survival.<br><br>
            <b>Tissue specific and common DNA methylation patterns.</b> Using the DNA methylation data across 54 human cell lines, we identified 35 276 local DNA methylation regions called local clusters of CpG sites (LCCSs). We constructed an LCCS co-methylation network and investigated the common DNA methylation patterns across all cell lines, which reveal two distinct groups in terms of their methylation level and genomic characteristics. We further detected diverse sets of cell lineage-specific high- and low-methylation patterns, which were depleted in promoter, CpG island (CGI) and repeat regions but enriched in gene body and non-CGI regions, especially the CGI shore regions. We discovered that the cell lineage-specific low-methylated LCCSs were enriched with functional transcriptional factor binding motif regions. Moreover, the detected cell line-specific high- and low-methylated patterns show distinct enrichments in cell line-specific chromatin states and functional relevance with the corresponding cell lines.<br><br>
            <b>Long non-coding RNA function annotation based on network model. </b>We develop a bi-colored network based global function predictor, long non-coding RNA global function predictor (‘lnc-GFP’), to predict probable functions for lncRNAs at large scale by integrating gene expression data and protein interaction data. The performance of lnc-GFP is evaluated on protein-coding and lncRNA genes. Cross-validation tests on protein-coding genes with known function annotations indicate that our method can achieve a precision up to 95%, with a suitable parameter setting. Among the 1713 lncRNAs in the bi-colored network, the 1625 (94.9%) lncRNAs in the maximum connected component are all functionally characterized. For the lncRNAs expressed in mouse embryo stem cells and neuronal cells, the inferred putative functions by our method highly match those in the known literature.<br><br>
            <img src ="images/Yang Xiaofei_research_3.png" alt = " " height="400" /><img src ="images/Yang Xiaofei_research_4.png" alt = " " height="400" align="right"/>
            </div>
            <div id = "publications">
                <h5>Publications</h5>
                <h6 style="font-size:20px;">2019</h6>
                1.	<b>Xiaofei Yang</b>, Wan-Ping Lee, Kai Ye, Charles Lee. One reference genome is not enough [J]. <b>Genome Biology</b>, 2019, 20(1): 104.<br>
                2.	Yongyong Kang, <b>Xiaofei Yang</b>, Jiadong Lin, Kai Ye. PVTree: A Sequential Pattern Mining Method for Alignment Independent Phylogeny Reconstruction. <b>Genes</b>, 2019, 10(2): 73.<br>
                <h6 style="font-size:20px;">2018</h6>
                3.	Li Guo*, Thilo Winzer*, <b>Xiaofei Yang*</b>, Yi Li*, Zemin Ning*, Zhesi He, Roxana Teodor, Ying Lu, Tim A. Bowser, Ian A. Graham†, Kai Ye†, The opium poppy genome and morphinan production, <b>Science</b>, 30 Aug 2018: eaat4096 DOI: 10.1126/science.aat4096（*:并列一作, †通讯作者)<br>
                4.	Kai Ye, Li Guo, <b>Xiaofei Yang</b>, et al, Split-Read Indel and Structural Variant Calling Using PINDEL, <b>Methods in molecular biology</b> (Clifton, N.J.) 1833:95-105, July 2018, DOI: 10.1007/978-1-4939-8666-8_7.<br>
                <h6 style="font-size:20px;">2017 and before</h6>
                5.	<b>Xiaofei Yang</b>, Lin Gao, Shihua Zhang. Comparative pan-cancer DNA methylation analysis reveals cancer common and specific patterns, <b>Briefings in Bioinformatics</b>, 2017, 18(5):761-773. <br>
                6.	Xiaoke Ma, Liang Yu, Peizhuo Wang, <b>Xiaofei Yang</b>. Discovering DNA methylation patterns for long non-coding RNAs associated with cancer subtypes. <b>Computational biology and chemistry</b>,2017, 69, 164-170.<br>
                7.	<b>Xiaofei Yang</b>, Xiaojian Shao, Lin Gao, et al. Comparative DNA methylation analysis to decipher common and cell type-specific patterns among multiple cell types. <b>Briefings in Functional Genomics</b>, 2016, 15(6):399-407. <br>
                8.	Feng Li, Lin Gao, Xiaoke Ma, <b>Xiaofei Yang</b>. Detection of driver pathways using mutated gene network in cancer . <b>Molecular BioSystems</b>, 2016,12, 2135-2141. <br>
                9.	<b>Xiaofei Yang</b>, Xiaojian Shao, Lin Gao, et al. Systematic DNA methylation analysis of multiple cell lines reveals common and specific patterns within and across tissues of origin. <b>Human Molecular Genetics</b>, 2015, 24(15): 4374-4384.<br> 
                10.	Hao Wu, Lin Gao, Feng Li, Fei Song, <b>Xiaofei Yang</b>, et al. Identifying overlapping mutated driver pathways by constructing gene networks in cancer. <b>BMC Bioinformatics</b>, 2015, 16(Suppl 5): S3. <br>
                11.	<b>Xiaofei Yang</b>, Lin Gao, Xingli Guo, et . A network based method for analysis of lncRNA-disease associations and prediction of lncRNAs implicated in diseases. <b>PLoS One</b>, 2014, 9(1): e87797. <br>
                12.	Xingli Guo, Lin Gao, Qi Liao, Hui Xiao, Xiaoke Ma, <b>Xiaofei Yang</b>, et al. Long non-coding RNAs function annotation: a global prediction method based on bi-colored networks. <b>Nucleic Acids Research</b>, 2013, 41(2): e35-e35. <br>
                13.	Hao Wu, Lin Gao, Jihua Dong, <b>Xiaofei Yang</b>. Detecting overlapping protein complexes by rough-fuzzy clustering in protein-protein interaction networks. <b>PLoS One</b>, 2014, 9(3): e91856. <br>
                14.	郭杏莉, 高琳*, 刘永轩, 党合萱, 王羽, <b>杨晓飞</b>, et al. 长非编码 RNA 生物特征研究与分析. <b>科学通报</b>, 2013, 58(27): 2779-2786.<br>
                15.	Xingli Guo, Lin Gao, Chunshui Wei, <b>Xiaofei Yang</b>, et al. A computational method based on the integration of heterogeneous networks for predicting disease-gene associations. <b>PLoS One</b>, 2011, 6(9): e34171. <br>
            </div>
            <div id = "awards">
            <h5>Awards and Grants</h5>
            <b>Grant:</b><br>
            1.	Development and transformation of biomarkers for precision medicine of hepatitis B associated liver cancer，No.2018ZX10302205，the National Science and Technology Major Project of China，2018.6 ~ 2020.12，Participant<br>
            2.	Network based method for discovering abnormal DNA methylation patterns, No. 61702406,  National Natural Science Foundation of China, 2018.1 ~ 2020.12，Principle Investigator<br>
            3.	Theory and methods for discovering cancer genomic patterns based on network model, No.61532014, National Natural Science Foundation of China，2016.1~2020.12        Participant
            <br>
            <b>Award:</b><br>
            2014.12 National scholarship for graduate students<br>
            2015.12 National scholarship for graduate students<br>
            <div id = "Professional">
            <h5>Professional Societies</h5>
            Reviewers of journals like Database, Neurocomputing, Briefings in Functional Genomics. Reviews for CAC2015 and CAC2018.
            </div>
    </div>
</div>
